### ISSUE TITLE
Wheel/Exe for Python 3.5

### ISSUE THREAD

**Issue by msklosak** [HexDecimal/libtcod-cffi#4]
There are currently no .whl or .exe installers for the latest Python version, 3.5.*. I have a 64-bit version of Python installed and trying to compile on Windows is a real pain. Updated installers would be a big help.

**Comment by HexDecimal** [comment]
Turns out Python 3.5 has a very different build environment than all of the earlier versions.  It's refusing to build at the moment and it might take a while to resolve the compiler issues I ran into.

I'm missing the 64-bit DLL's for compiling for 64-bit on Windows.  The version of libtcod that I'm using didn't support 64-bit very well, so at the moment it only compiles for 64 on Linux and that variant might not be bug-free.  I'll try too build for 64-bits after getting the 32-bit version working.

For now I can only recommend using the 32-bit version of Python 3.4 or trying out PyPy.
